lib/runTestFixtures.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+path = require('path');
+const defineTest = require('jscodeshift/src/testUtils').defineTest;
+
+/**
+ * Runs all testfixtures in folder with rewriter script
+ */
+function runTestFixtures() {
```

Comment:
Your style here seems to be inconsistent with elsewhere, where you chose `const x = () => {}` for all your functions. https://eslint.org/docs/rules/func-style can enforce one or the other.